usr/src/uts/common/sys/file.h

```diff
@@ -125,6 +125,11 @@ typedef struct fpollinfo {
  * Private interface for lx O_PATH|O_NOFOLLOW emulation for symlinks.
  */
 #define	__FLXPATH	0x80000000
+/*
+ * Private interface for lx fstatat(AT_NO_AUTOMOUNT) emulation.
+ * Since usage is disjoint, the __FLXPATH bit is re-used.
+ */
+#define	__FLXNOAUTO	0x80000000
```

Comment:
Reusing the flag bit won't fubar anything else?  Seems fishy...

Reply:
At the very least the comment needs to be a little more verbose about why we can reuse the bit...